topics/ecology/tutorials/Obitools-metabarcoding/tutorial.md

```diff
@@ -0,0 +1,537 @@
+>    - *"Input sequences file"*: `obigrep output file`
+>    - *"Maximum numbers of differences between two variant sequences (default: 1)"*: `1`
+>    -  *"Threshold ratio between counts (rare/abundant counts) of two sequence records so that the less abundant one is a variant of the more abundant (default: 1, i.e. all less abundant sequences are variants)"*: `0.05`
+>    - *"Do you want to select only sequences with the head status in a least one sample?"*: `Yes`
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > As a final denoising step, using the obiclean program, we keep the head sequences that are sequences with no variants with a count greater than 5% of their own count
+>    {: .comment}
+>
+{: .hands_on}
+
+
+## Taxonomic assignment of sequences with **NCBI BLAST+ blastn**
+
+> ### {% icon hands_on %} Hands-on: Search nucleotide database with nucleotide query sequence(s) from OBITools treatments
+>
+> 1. {% tool [NCBI BLAST+ blastn](toolshed.g2.bx.psu.edu/repos/devteam/ncbi_blast_plus/ncbi_blastn_wrapper/2.10.1+galaxy0) %} with the following parameters:
+>    - *"Nucleotide query sequence(s)"*: `obiclean output file`
+>    - *"Subject database/sequences"*: `FASTA file from your history`
+>    - *"Nucleotide FASTA subject file to use instead of a database"*: `db_v05_r117`
+>    - *"Set expectation value cutoff"*: `0.0001`
+>    - *"Output format"*: `Tabular (extended 25 columns)`
+>    - *"Advanced Options"*: `Show Advanced Options`
+>    - *"Maximum hits to consider/show": `1`
+>
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > Here we directly use the `db_v05_r117` fasta file proposed on the [original obitools tutorial](https://pythonhosted.org/OBITools/wolves.html#step-by-step-analysis). One can mention you can create such a fasta file using same obitools workflow describe before (using obigrep/obiuniq/obigrep/obiannotate) on downloaded EMBL datrabases and taxonomy treated by obitools ecoPCR tool. 
+>    {: .comment}
+>
+{: .hands_on}
+
+
+## Filter database and query sequences by ID to re associate informations with **Filter sequences by ID**
+
+> ### {% icon hands_on %} Hands-on: Filter Blast results
+>
+> 1. {% tool [Filter sequences by ID](toolshed.g2.bx.psu.edu/repos/peterjc/seq_filter_by_id/seq_filter_by_id/0.2.7) %} with the following parameters:
+>    - *"Sequence file to be filtered"*: `db_v05_r117`
+>    - *"Filter using the ID list from"*: `tabular file`
+>    - *"Tabular file containing sequence identifiers"*: `megablast on obiclean output`
+>    - *"Column(s) containing sequence identifiers"*: `Column 2`
+>    - *"Output positive matches, negative matches, or both?"*: `just positive match`
+>
+>
+>    > ### {% icon comment %} Comment
+>    >
+>    > This tool allows you to re-associate all the reference sequences information, notably the `species_name` one so you can see which species are potentially seen on the sample.
+>    {: .comment}
```

Comment:
I think that both comment blocks should be moved to the beginning in order to explain the objective of this step.

Reply:
yes! Thank you, done!